item-api/src/main/java/com/example/auction/item/api/UpdateItemResult.java

```diff
@@ -0,0 +1,31 @@
+package com.example.auction.item.api;
+
+
+public final class UpdateItemResult {
```

Comment:
I thought of responding  with a `Location:` header but that'd be 2 round trips. I'll send the item back. 👍